src/EA.Weee.Web/Areas/Admin/Controllers/AatfController.cs

```diff
@@ -74,7 +76,8 @@ public async Task<ActionResult> Details(Guid id)
                     OrganisationString = GenerateSharedAddress(aatf.Organisation.BusinessAddress),
                     AssociatedAatfs = associatedAatfs,
                     AssociatedSchemes = associatedSchemes,
-                    SubmissionHistory = submissionHistory
+                    SubmissionHistory = submissionHistory,
+                    ComplianceYearList = years
```

Comment:
Update the DetailsGet_GivenValidAatfId_MapperShouldBeCalled test to check the mapper s called with this list

Reply:
Added